package.json

```diff
@@ -32,7 +32,7 @@
     "size:why": "pnpm size:snapshot --analyze --accurateBundles",
     "tc": "node test/cli.js",
     "test": "node scripts/test.mjs",
-    "test:coverage": "cross-env NODE_ENV=test TZ=UTC BABEL_ENV=coverage nyc mocha --exclude '**/node_modules/**' && nyc report -r lcovonly",
+    "test:coverage": "cross-env NODE_OPTIONS=--max-old-space-size=4096 NODE_ENV=test TZ=UTC BABEL_ENV=coverage nyc mocha --exclude '**/node_modules/**' && nyc report -r lcovonly",
```

Comment:
@LukasTy How did you manage to make the `test_unit` pass? It failed even when I added this flag here

Reply:
Did it? I saw them passing even before bumping jsdom... 🤷 
https://app.circleci.com/pipelines/github/mui/mui-x?branch=pull%2F12880

Reply:
Ahh, I think I didn't wait for the result after my last commit, and for some reason assumed it failed 😅
All good then 👍🏻

Reply:
P.S. Here is my comment comparing your commits behavior on the system resources. 😉 
https://github.com/mui/mui-x/pull/12880#issuecomment-2136584644